OpenSim/Simulation/Wrap/WrapDoubleCylinderObst.h

```diff
@@ -107,25 +95,17 @@ OpenSim_DECLARE_CONCRETE_OBJECT(WrapDoubleCylinderObst, WrapObject);
 #ifndef SWIG
     WrapDoubleCylinderObst& operator=(const WrapDoubleCylinderObst& aWrapDoubleCylinderObst);
 #endif
-    void copyData(const WrapDoubleCylinderObst& aWrapDoubleCylinderObst);
-
-    double getRadius() const { return _radiusUcyl; }
-    void setRadius(double aRadius) { _radiusUcyl = aRadius; }
-    double getLength() const { return _length; }
-    void setLength(double aLength) { _length = aLength; }
-    //WrapDirectionEnum getWrapDirection() const { return _wrapUcylDirection; }
-    int getWrapDirection() const { return (int)_wrapUcylDirection; }
```

Comment:
It seems redundant to me since they are created via the macro. Also, there is nowhere in the code where this is hard coded.
But I understand, that some backward compatibility may needs this..

Reply:
I was thinking also, these getter and setter are highly misleading too.. Since they only modify one of the wrapping... 
I think these getter and setter are actually a copy-paste from WrapCylinderObst.h where the coder decided to keep them there without actually really think if he needed them. I highly doubt anyone used these before since they probably produce some kind of unexpected behavior. 

I think we can get rid of these without much of worry... Anyway, for now, I implemented them, but if you prefer, I will remove them!